DocumentedExamples/SeaIce_Obs_Model_Compare.ipynb

```diff
@@ -0,0 +1,5185 @@
+{
```

Comment:
<p>why the the very low values at 0.25° for the final year? Is this an incomplete year?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
Nice work Anton - interesting that ERA5 isn't a magic bullet but improves some things.
Thanks for doing this with the latest tools - it provides a good example to build from.

I only had a few comments. You might also consider using https://black.readthedocs.io/ or similar to give it a more uniform code style.

Reply:
<p>I will drop the last year as its incomplete (they end in July). Should we cull the start years too as its in spin-up ?</p>

Reply:
<p>OK to include the spinup here but I'd exclude the spinup (and final year) from the climatological cycle</p>